functions/man.fish

```diff
@@ -1,3 +1,14 @@
 function man --wraps man -d "Run man with added colors"
-    cless /usr/bin/man $argv
+    set --local --export MANPATH $MANPATH
+
+    if test -z "$MANPATH" && set path (command man -p 2>/dev/null)
```

Comment:
What is `command man -p 2>/dev/null` supposed to return? I looked up BSD and Linux manuals and -p is supposed to be used to specify preprocessors.

Reply:
That's just [what fish do](https://github.com/fish-shell/fish-shell/blob/555af37616893160ad1afb208a957d6a01a7a315/share/functions/man.fish#L15). I don't feel like copying the entire Fish function...